thread-manager/examples/core_contention_basics.rs

```diff
@@ -48,11 +48,12 @@ fn main() -> anyhow::Result<()> {
         println!("Running {exp}");
         let mut conffile = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
         conffile.push(exp);
-        let conffile = std::fs::File::open(conffile)?;
-        let cfg: RuntimeManagerConfig = serde_json::from_reader(conffile)?;
+        let mut buf = String::new();
+        std::fs::File::open(conffile)?.read_to_string(&mut buf)?;
+        let cfg: RuntimeManagerConfig = toml::from_str(&buf)?;
         //println!("Loaded config {}", serde_json::to_string_pretty(&cfg)?);
```

Comment:
I would use log / tracing crate instead of print

Reply:
done